remix/app/repositories/user.server.test.ts

```diff
@@ -0,0 +1,55 @@
+import type { User } from "@prisma/client";
+import { prismaMock } from "../tests/setup-test-env";
+import UserRespository from "../repositories/user.server"
+
+describe("Test user model", () => {
+    test("Create User", async () => {
+        const user: User = {
+            name: "Beckham",
+            email: "phisith@nimblehq.co",
+            avatarUrl: "weofkwepofkwefokwepfokwpo",
+            createdAt: new Date(),
+            id: 1
```

Comment:
Shouldn't this ID be created automatically?

Reply:
Yes, my prisma will auto generate the id.  For the reason that I declare a id here is because Typescript is force me to do that, I think to solve this issues is better to Generate Mockup data Library like `Faker.js`, like you have mention above.

Reply:
Fixed.